packages/sirius-web/backend/sirius-web-sample-application/Dockerfile

```diff
@@ -0,0 +1,6 @@
+FROM eclipse-temurin:17-jre
+COPY target/sirius-web-sample-application-2022.9.8.jar ./app.jar
```

Comment:
`sirius-web-sample-application-*.jar` instead of a hard-coded version number?

Reply:
It creates an issue with the presence of a `-source.jar`, that was the quick and dirty solution because I'm too lazy to tell `COPY` how to copy `sirius-web-sample-application-2022.9.8.jar` but not `sirius-web-sample-application-2022.9.8-sources.jar`. If you know how to do it easily, I would gladly like to know 😄

Reply:
From a regular shell, I'd do [like this](https://github.com/eclipse-sirius/sirius-web/blob/master/.github/workflows/build-sample-app.yml#L56), but Docker' `COPY` is more limited and [uses Go's matcher](https://docs.docker.com/engine/reference/builder/#copy). Let's keep it that way.